src/components/interaction-designer/clipboard/ClipboardRoot.vue

```diff
@@ -69,16 +86,37 @@ export default {
   created() {
     this.initializeFlowRunner()
   },
+  computed: {
+    isFocused(index) {
```

Comment:
if this isn't necessary for bug workaround, I'd suggest we use directly `isBlockFocused(index)`.

Reply:
You are right! It wasn't necessary. I removed it and used isBlockFocused directly